discord/flags.py

```diff
@@ -401,15 +401,23 @@ def suppress_notifications(self):
         .. versionadded:: 2.4
         """
 
-        return 4096
+        return 1 << 12
 
     @flag_value
     def is_voice_message(self):
         """:class:`bool`: Returns ``True`` if this message is a voice message.
 
         .. versionadded:: 2.5
         """
-        return 8192
+        return 1 << 13
+
+    @flag_value
+    def is_components_v2(self):
+        """:class:`bool`: Returns ``True`` if this message has v2 components. This flag disables sending `content` and `embeds`.
```

Comment:
```suggestion
        """:class:`bool`: Returns ``True`` if this message has v2 components. This flag disables sending `content` and `embeds`.
```
also `embed`, right?

Reply:
yes on embed, but am i insane or does that diff mean nothing

Reply:
the diff is messed up. add `embed` (singular)